tests/integration/internal/constants.go

```diff
@@ -22,6 +22,10 @@ const (
 	LogsGeneratorNamespace = "logs-generator"
 	LogsGeneratorName      = "logs-generator"
 	LogsGeneratorImage     = "sumologic/kubernetes-tools:2.9.0"
+
+	MultilineLogsNamespace   = "default"
```

Comment:
Can we make it a bit more concrete/unique? So that in case we need to debug we don't have logs from other stuff mixed in the `default` namespace? We can call it e.g. `multiline-tests` or something like that

Reply:
ok